src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -36,10 +36,17 @@ class FileMetadataService(fileMetadataRepository: FileMetadataRepository,
 }
 
 object FileMetadataService {
+  case class StaticMetadata(name: String, value: String)
   val SHA256ClientSideChecksum = "SHA256ClientSideChecksum"
   val ClientSideOriginalFilepath = "ClientSideOriginalFilepath"
   val ClientSideFileLastModifiedDate = "ClientSideFileLastModifiedDate"
   val ClientSideFileSize = "ClientSideFileSize"
+  val RightsCopyright: StaticMetadata = StaticMetadata("RightsCopyright", "Crown Copyright")
```

Comment:
I think it's worth adding a code comment explaining the reason for these fields. It's a business decision, so it's not obvious from the code why they need to be hard-coded.

Reply:
I've put a comment in saying more or less what it says on the Jira ticket.

Reply:
Heh, I don't think that saying "it was a business decision" helps the person reading it understand the reason. 🙂 

I meant more like:

> Save default values for these properties because TDR currently only supports records which are Open, in English, etc. Users agree to these conditions at a consignment level, so it's OK to safe these as defaults for every file. They need to be saved so they can be included in the export package. The defaults may be removed in future once we let users upload a wider variety of records.

Reply:
That's a lot better thanks :+1: